### PR TITLE
Return_the_coordinates.rb

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: Return_the_coordinates.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  ["O", "O", "O", "O"]
+]) ➞ [5, 1]
+Notes
+The given array will always be a square.
+Rows and columns are 1-indexed (not zero-indexed).
+=end
+
+def where_is_waldo(array)
+   coordinates = []
+   value = array[0][0]
```
Comment (danielpaul) [inline Return_the_coordinates.rb]: there is an edge case that this code will not work for. Try and solve for this:

```
where_is_waldo([
   ["P", "O", "O", "O"],
   ["O", "O", "O", "O"],
   ["O", "O", "O", "O"],
   ["O", "O", "O", "O"],
   ["O", "O", "O", "O"],
   ["O", "O", "O", "O"]
 ]) ➞ [1, 1]
```

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: Return_the_coordinates.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+The given array will always be a square.
+Rows and columns are 1-indexed (not zero-indexed).
+=end
+
+def where_is_waldo(array)
+   coordinates = []
+   array.each_with_index do |i, index_a|
+    i.find_all.each_with_index { |e, index_b|  coordinates.push(index_a + 1, index_b + 1) if i.count(e) == 1 }
+   end
+   coordinates
```
Comment (danielpaul) [inline Return_the_coordinates.rb]: can you write the algorithm here in english? not clear of what you are doing exactly to find it.

Comment (Anishluke92) [reply Return_the_coordinates.rb]: i thought of using those inbuilt function and finding the answer. but actually it was not that good a decision

Comment (Anishluke92) [reply Return_the_coordinates.rb]: But the proper code is been uploaded

**Pull request merged**